settings.py

```diff
@@ -79,6 +79,25 @@
 # Set to True if we're allowed to use X-SENDFILE.
 XSENDFILE = False
 
+
+if DEBUG:
+    # To activate the Django debug toolbar.
+    INSTALLED_APPS += (
+        'debug_toolbar',
+    )
+    # Prevent DDT from patching the settings.
+    DEBUG_TOOLBAR_PATCH_SETTINGS = False
+
+    MIDDLEWARE_CLASSES += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+
+    def show_toolbar_callback(request):
+        return True
+
+    DEBUG_TOOLBAR_CONFIG = {
+        'SHOW_TOOLBAR_CALLBACK': 'local_settings.show_toolbar_callback'
```

Comment:
fwiw it looks like that was reversed at some point so that you can still use a callable https://github.com/django-debug-toolbar/django-debug-toolbar/commit/cb428ce6b218f83e45d362d16d1a33143b8e2701